lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
> WebTorrent.app and webtorrent-hybrid should probably stop adding trackers to added files as well 

So it will be kept removed for now to avoid adding 3rd party endpoints to the torrents.